src/audio/eq_iir/eq_iir.c

```diff
@@ -457,19 +496,18 @@ static int eq_iir_init_coef(struct sof_eq_iir_config *config,
 	int j;
 	int s;
 
-	comp_cl_info(&comp_eq_iir, "eq_iir_init_coef(), response assign for %u channels, %u responses",
-		     config->channels_in_config,
-		     config->number_of_responses);
+	comp_info(dev, "eq_iir_init_coef(), response assign for %u channels, %u responses",
```

Comment:
@ranj063 Have "mod->dev" here. We had this prior to the module conversion, gcc was upgraded during this time and now your changes are bringing the warnings back.

Reply:
Thanks @kv2019i